.travis.yml

```diff
@@ -19,17 +19,16 @@ matrix:
 
       # processing module test
     - sudo: false
-      install: mvn install -q -ff -DskipTests -B
-      script: mvn test -B -Pparallel-test -Dmaven.fork.count=2 -pl processing
+      install: echo "MAVEN_OPTS='-Xmx3000m'" > ~/.mavenrc && mvn install -ff -DskipTests -B
+      script: echo "MAVEN_OPTS='-Xmx512m'" > ~/.mavenrc && mvn test -B -Pparallel-test -Dmaven.fork.count=2 -pl processing
 
       # non-processing modules test
     - sudo: false
-      install: mvn install -q -ff -DskipTests -B
```

Comment:
```-q``` option isn't necessary? There are too many logs again, so that we may not be able to check the reason on failures. (https://travis-ci.org/druid-io/druid/jobs/272346011)

Reply:
I added back the `-q` option. I had removed it in testing since it was making it difficult to see where the travis job was crashing.